pkgs/applications/networking/xtreme-download-manager/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, jdk11
+, maven
+, makeWrapper
+}:
+
+let
+  mvn2nix = import
+    (fetchFromGitHub {
+      owner = "fzakaria";
+      repo = "mvn2nix";
+      rev = "bc86f650d80ce9d29c376d6955ed175cba87915e";
+      sha256 = "0lzy1l208sxrs83y6l7dlrbmg1q3y75gw4fxy3wpzjhsxlycv3xj";
+    }) { };
+  mavenRepository =
+   mvn2nix.buildMavenRepositoryFromLockFile { file = ./mvn2nix-lock.json; };
+  source = fetchFromGitHub {
+    owner = "subhra74";
+    repo = "xdm";
+    rev = "7.2.11";
+    sha256 = "1p2mzaaig7fxk3gsk7r5shl1hijfz6bjlhmkqxgipywr1q1f67fb"; 
+  };
+in stdenv.mkDerivation rec {
+  pname = "xtreme-download-manager";
+  version = "7.2.11";
+  name = "${pname}-${version}";
```

Comment:
I think it's because it needs to access the name variable in installPhase. But you're right, I've replaced all occurences of ${name} with ${pname}-${version} and removed this line.